components/docs-chef-io/content/automate/install_os.md

```diff
@@ -0,0 +1,380 @@
++++
```

Comment:
Replace the existing content of Elasticsearch with OpenSearch on the same exiting page. Drop this new page.